src/it/scala/org/apache/spark/sql/snowflake/SFTestSessionBase.scala

```diff
@@ -30,6 +31,8 @@ trait SFTestSessionBase extends IntegrationEnv {
 
   override def beforeAll(): Unit = {
     super.beforeAll()
+    // Run ported test in WARN level to avoid large log volume
+    reconfigureLogFile(PORTED_TEST_LOG4J_PROPERTY)
```

Comment:
Why not just use `Logger.setLevel()`?

Reply:
Logger.setLevel() is NOT a static function. It needs to be called for a logger object. So it can't change logging level for all loggers.

Reply:
`Configurator.setAllLevels()` 

Ref: https://stackoverflow.com/questions/23434252/programmatically-change-log-level-in-log4j2

Reply:
That page is for log4j2 which is different to log4j.
I have tried more research. I didn't find better way to set log level in program.